modules/react/loading-dots/stories/LoadingDots.mdx

```diff
@@ -2,6 +2,8 @@ import {ExampleCodeBlock} from '@workday/canvas-kit-docs';
 import {Basic} from './examples/Basic';
```

Comment:
It won't let me do it, but let's import `SymbolDoc` from `@workday/canvas-kit-docs`